.github/workflows/regression.yml

```diff
@@ -58,9 +58,10 @@ jobs:
         run: |
           if [[ "${{ github.event_name }}" == "pull_request" ]]; then
             echo "SHOULD_RUN=false" >> $GITHUB_OUTPUT
-          elif [[ "${{ github.event_name }}" != "pull_request_review" &&
-                  "${{ !contains(github.event.pull_request.labels.*.name, 'ci-condition: skip-regression') }}" == "true" ]]; then
-            # The small caveat here is that we still want to run the suite via a review comment.
+          elif [[ "${{ github.event_name }}" == "pull_request_review" ]]; then
```

Comment:
Ah, interesting, maybe `workflow_call` preserves the event that triggered the parent workflow. I do see it being used in the workflow run you linked to.